src/System.Windows.Forms.Primitives/src/Interop/ComCtl32/Interop.TVN.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal static partial class Interop
+{
+    internal static partial class ComCtl32
+    {
+        public enum TVN : int
+        {
+            FIRST = -400,
```

Comment:
Please keep it aligned with Win32 definitions
```suggestion
            FIRST = 0 - 400,
```

We've done the same done for `LVN`